tests/system/python/packages/test_multiple_assets.py

```diff
@@ -30,20 +30,20 @@
 FLEDGE_ROOT = os.environ.get('FLEDGE_ROOT')
 BENCHMARK_SOUTH_SVC_NAME = "BenchMark #"
 ASSET_NAME = "random_multiple_assets"
-PER_BENCHMARK_ASSET_COUNT = 150
+# PER_BENCHMARK_ASSET_COUNT = 100
 AF_HIERARCHY_LEVEL = "multipleassets/multipleassetslvl2/multipleassetslvl3"
 
 
 @pytest.fixture
-def reset_fledge(wait_time):
```

Comment:
Why? 

You can do 2*wait time if needed but keep it based on passed value that allows to pass any value we want